### ISSUE TITLE
Plugin could not be activated because it triggered a fatal error

### ISSUE THREAD

**Issue by NinjaDoll** [bueltge/multisite-global-media#85]
Error Message:

_Parse error: syntax error, unexpected ':', expecting '{' in /home/cydco2/public_html/wp-content/plugins/multisite-global-media-master/multisite-global-media.php on line 49_

WP Build 5.1.1

**Comment by bueltge** [comment]
Please check your php version, I think is to small to our requirements.

**Comment by NinjaDoll** [comment]
Running 5.6.40, servers were just upgraded about two months ago.

**Comment by widoz** [comment]
@NinjaDoll the plugin require at least php 7.0. You should see a message in the admin page such as this one https://github.com/bueltge/multisite-global-media/blob/master/multisite-global-media.php#L82

**Comment by NinjaDoll** [comment]
That message didn't come up during the attempt at install but thanks, there's no way my provider's going to upgrade to 7.0. Bummer! Thanks for responding!

**Comment by widoz** [comment]
Sorry to hear that @NinjaDoll but I would suggest you to have a look at an host that allow you to switch to php7 since it's more performant and solve a lot of php bugs.

Btw, thanks for having reported the problem I discovered why you didn't get the message in the admin and instead a php error.

I created a pull request #87 so may be in future other users will get notified without WOSD when they'll try to activate the plugin with an incompatible php version.
